upgrade_test.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-
+import sys
```

Comment:
why added `sys` here?

Reply:
you are right, leftover from previous trials, removing